src/Symfony/Component/HttpFoundation/ResponseHeaderBag.php

```diff
@@ -63,8 +63,12 @@ public function __construct(array $headers = array())
     public function __toString()
     {
         $cookies = '';
-        foreach ($this->getCookies() as $cookie) {
-            $cookies .= 'Set-Cookie: '.$cookie."\r\n";
+        try {
+            foreach ($this->getCookies() as $cookie) {
+                $cookies .= 'Set-Cookie: '.$cookie."\r\n";
+            }
+        } catch (\InvalidArgumentException $wrongFormat) {
+            trigger_error($wrongFormat->getMessage(), E_USER_ERROR);
```

Comment:
Same here: no exception is going to be thrown, the format is right and can't be wrong.